articles/active-directory-b2c/find-help-open-support-ticket.md

```diff
@@ -41,7 +41,7 @@ If you're unable to find answers by using self-help resources, you can open an o
 
 1. Sign in to the [Azure portal](https://portal.azure.com).
 
-1. If you have access to multiple tenants, select the **Settings** icon in the top menu to switch to your Azure AD B2C tenant from the **Directories + subscriptions** menu.
+1. If you have access to multiple tenants, select the **Settings** icon in the top menu to switch to your Entra Workforce tenant from the **Directories + subscriptions** menu.  Currently, technical support cases cannot be created directly from an Azure Active Directory Business to Consumer (B2C) tenant.
```

Comment:
```suggestion
1. If you have access to multiple tenants, select the **Settings** icon in the top menu to switch to your Microsoft Entra tenant from the **Directories + subscriptions** menu. Currently, you can't submit support cases directly from your Azure AD B2C tenant.
```

Reply:
@jasonfritts Please check this suggestion, then commit.

Reply:
#sign-off

Reply:
@jasonfritts you need to commit the suggested changes, then I will sign off the PR. 

Thanks.